README.md

```diff
@@ -1 +1,166 @@
-# Catalog-of-my-things kl
\ No newline at end of file
+<div align="center">
+  <h1>Catalog_of_my_things</h1>
+</div>
+
+
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Catalog_of_my_things ](#-Catalog_of_my_things-)
+  - [🛠 Built With ](#-built-with-)
+    - [Key Features ](#key-features-)
+  - [🤯 Walk through video ](#-walk-through-video-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Catalog_of_my_things <a name="about-project"></a>
+
+> Catalog of my things Project is a Ruby-based console application designed to allows users to organize and store data about their personal catalog of items. With this application, you can easily manage various types of collections, such as books, music albums, movies, or any other items you wish to catalog.
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Ruby
+- PSQL
+
+<!-- Features -->
+### Key Features <a name="key-features"></a>
+
+- Item Management: Create and view and delete items in your catalog. 
+
+- Data Persistence: The application ensures data persistence by storing your catalog data in a JSON file. This allows you to access and manage your catalog across different sessions, ensuring that your data is preserved.
+
+- User-Friendly Interface: The console-based interface provides a straightforward and intuitive user experience. It allows you to navigate through menus, input data, and interact with your catalog using simple commands and prompts
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## Walkthrough video <a name="walk-through"></a>
+
+ [video](https://drive.google.com/file/d/1TYLRgO1Jb216s9gm3s-mEgsSvbxbpiwH/view?usp=drive_link)
```

Comment:
- [ ] Kindly record the video again, It has to be shorter, less than 5 minutes. It's not acceptable and it's not professional to see in a capstone presentation a person walking shirtless and one of the students it's not present for a couple of minutes in the video. Remember that this video could be seen by your future recruiters, and it's really important to look at professionals 🎩

Reply:
thank you for your feedback. we would surely work on that